plugins/modules/iam_policy.py

```diff
@@ -92,11 +92,72 @@
 """
 
 RETURN = r"""
```

Comment:
Possible followup - since the other modules all bundle this under a single return value - it's probably worth being consistent and doing the same for iam_policy.